README.md

```diff
@@ -1,59 +1,76 @@
-# \<link[rel="lazy-import"]\> [![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+# \<link rel="lazy-import"\>
 
-Declarative lazy HTML imports
+[![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
```

Comment:
I would like to propose update to the official Travis image and sort like most of the PolymerElements:

```
[![Build Status](https://travis-ci.org/Polymer/lazy-imports.svg?branch=master)](https://travis-ci.org/Polymer/lazy-imports)

# \<link rel="lazy-import"\>

This repo implements [...]
```